tb_device_mqtt.py

```diff
@@ -311,3 +311,50 @@ def subscribe_to_attribute(self, key, callback):
 
     def wait_for_msg(self):
         self._client.wait_msg()
+class ProvisionClient:
+    PROVISION_REQUEST_TOPIC = "/provision/request"
+    PROVISION_RESPONSE_TOPIC = "/provision/response"
+
+    def __init__(self, host, port, provision_request):
```

Comment:
What is the `provision_request` argument?
If it is dict type object, what keys should it contain?